Still can't run kubernetes tests in Vagrant environment

# Description of problem

With #3781 fixed we get further attempting to run the kubernetes tests in the Vagrant environment.  But.. there's another problem.

Setup is:

```
$ vagrant up ubuntu
[...]
$ vagrant ssh ubuntu
[...]
vagrant@ubuntu2004:~/go/src/github.com/kata-containers/tests$ make kubernetes
```

# Expected result

Kubernetes tests run to completion.

# Actual result

```
vagrant@ubuntu2004:~/go/src/github.com/kata-containers/tests$ make kubernetes
bash -f .ci/install_bats.sh
Install BATS from sources
package github.com/bats-core/bats-core: no Go files in /home/vagrant/go/src/github.com/bats-core/bats-core
~/go/src/github.com/bats-core/bats-core ~/go/src/github.com/kata-containers/tests
Installed Bats to /usr/bin/bats
~/go/src/github.com/kata-containers/tests
bash -f integration/kubernetes/run_kubernetes_tests.sh
~/go/src/github.com/kata-containers/tests/integration/kubernetes ~/go/src/github.com/kata-containers/tests
Build custom stress image
~/go/src/github.com/kata-containers/tests/integration/kubernetes/runtimeclass_workloads/stress ~/go/src/github.com/kata-containers/tests/integration/kubernetes
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM quay.io/libpod/ubuntu
latest: Pulling from libpod/ubuntu
da7391352a9b: Pull complete 
14428a6d4bcd: Pull complete 
2c2d948710f2: Pull complete 
Digest: sha256:c95a8e48bf88e9849f3e0f723d9f49fa12c5a00cfc6e60d2bc99d87555295e4c
Status: Downloaded newer image for quay.io/libpod/ubuntu:latest
 ---> f643c72bc252
Step 2/2 : RUN apt-get -y update &&     apt-get -y upgrade &&     apt-get -y install stress
 ---> Running in 746aa8bf0216
Get:1 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:3 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [792 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
Get:7 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [30.6 kB]
Get:8 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [1034 kB]
Get:9 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [488 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]
Get:12 http://archive.ubuntu.com/ubuntu focal/multiverse amd64 Packages [177 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1471 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [33.8 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [534 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1065 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [2668 B]
Get:18 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [6324 B]
Fetched 18.9 MB in 19s (1012 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
The following packages will be upgraded:
  apt base-files gcc-10-base gpgv libapt-pkg6.0 libc-bin libc6 libgcc-s1
  libgnutls30 libhogweed5 liblz4-1 libnettle7 libp11-kit0 libpam-modules
  libpam-modules-bin libpam-runtime libpam0g libprocps8 libseccomp2 libstdc++6
  libsystemd0 libudev1 libzstd1 login passwd procps tar ubuntu-keyring
28 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 10.2 MB of archives.
After this operation, 95.2 kB disk space will be freed.
Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gcc-10-base amd64 10.3.0-1ubuntu1~20.04 [20.2 kB]
Get:2 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libgcc-s1 amd64 10.3.0-1ubuntu1~20.04 [41.8 kB]
Get:3 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libstdc++6 amd64 10.3.0-1ubuntu1~20.04 [501 kB]
Get:4 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libc6 amd64 2.31-0ubuntu9.2 [2715 kB]
Get:5 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 base-files amd64 11ubuntu5.4 [60.6 kB]
Get:6 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 login amd64 1:4.8.1-1ubuntu5.20.04.1 [220 kB]
Get:7 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 tar amd64 1.30+dfsg-7ubuntu0.20.04.1 [240 kB]
Get:8 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libc-bin amd64 2.31-0ubuntu9.2 [639 kB]
Get:9 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 liblz4-1 amd64 1.9.2-2ubuntu0.20.04.1 [56.1 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libsystemd0 amd64 245.4-4ubuntu3.11 [270 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libudev1 amd64 245.4-4ubuntu3.11 [78.4 kB]
Get:12 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libzstd1 amd64 1.4.4+dfsg-3ubuntu0.1 [237 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libapt-pkg6.0 amd64 2.0.6 [835 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpgv amd64 2.2.19-3ubuntu2.1 [199 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 ubuntu-keyring all 2020.02.11.4 [22.1 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libnettle7 amd64 3.5.1+really3.5.1-2ubuntu0.2 [115 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libhogweed5 amd64 3.5.1+really3.5.1-2ubuntu0.2 [138 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libp11-kit0 amd64 0.23.20-1ubuntu0.1 [188 kB]
Get:19 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libgnutls30 amd64 3.6.13-2ubuntu1.6 [828 kB]
Get:20 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libseccomp2 amd64 2.5.1-1ubuntu1~20.04.1 [42.9 kB]
Get:21 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 apt amd64 2.0.6 [1296 kB]
Get:22 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpam0g amd64 1.3.1-5ubuntu4.2 [55.4 kB]
Get:23 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpam-modules-bin amd64 1.3.1-5ubuntu4.2 [41.2 kB]
Get:24 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpam-modules amd64 1.3.1-5ubuntu4.2 [260 kB]
Get:25 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpam-runtime all 1.3.1-5ubuntu4.2 [37.3 kB]
Get:26 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 passwd amd64 1:4.8.1-1ubuntu5.20.04.1 [799 kB]
Get:27 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libprocps8 amd64 2:3.3.16-1ubuntu2.2 [33.0 kB]
Get:28 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 procps amd64 2:3.3.16-1ubuntu2.2 [232 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 10.2 MB in 13s (780 kB/s)
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../gcc-10-base_10.3.0-1ubuntu1~20.04_amd64.deb ...
Unpacking gcc-10-base:amd64 (10.3.0-1ubuntu1~20.04) over (10.2.0-5ubuntu1~20.04) ...
Setting up gcc-10-base:amd64 (10.3.0-1ubuntu1~20.04) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libgcc-s1_10.3.0-1ubuntu1~20.04_amd64.deb ...
Unpacking libgcc-s1:amd64 (10.3.0-1ubuntu1~20.04) over (10.2.0-5ubuntu1~20.04) ...
Setting up libgcc-s1:amd64 (10.3.0-1ubuntu1~20.04) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libstdc++6_10.3.0-1ubuntu1~20.04_amd64.deb ...
Unpacking libstdc++6:amd64 (10.3.0-1ubuntu1~20.04) over (10.2.0-5ubuntu1~20.04) ...
Setting up libstdc++6:amd64 (10.3.0-1ubuntu1~20.04) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libc6_2.31-0ubuntu9.2_amd64.deb ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Unpacking libc6:amd64 (2.31-0ubuntu9.2) over (2.31-0ubuntu9.1) ...
Setting up libc6:amd64 (2.31-0ubuntu9.2) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../base-files_11ubuntu5.4_amd64.deb ...
Unpacking base-files (11ubuntu5.4) over (11ubuntu5.2) ...
Setting up base-files (11ubuntu5.4) ...
Installing new version of config file /etc/issue ...
Installing new version of config file /etc/issue.net ...
Installing new version of config file /etc/lsb-release ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../login_1%3a4.8.1-1ubuntu5.20.04.1_amd64.deb ...
Unpacking login (1:4.8.1-1ubuntu5.20.04.1) over (1:4.8.1-1ubuntu5.20.04) ...
Setting up login (1:4.8.1-1ubuntu5.20.04.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../tar_1.30+dfsg-7ubuntu0.20.04.1_amd64.deb ...
Unpacking tar (1.30+dfsg-7ubuntu0.20.04.1) over (1.30+dfsg-7) ...
Setting up tar (1.30+dfsg-7ubuntu0.20.04.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libc-bin_2.31-0ubuntu9.2_amd64.deb ...
Unpacking libc-bin (2.31-0ubuntu9.2) over (2.31-0ubuntu9.1) ...
Setting up libc-bin (2.31-0ubuntu9.2) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../liblz4-1_1.9.2-2ubuntu0.20.04.1_amd64.deb ...
Unpacking liblz4-1:amd64 (1.9.2-2ubuntu0.20.04.1) over (1.9.2-2) ...
Setting up liblz4-1:amd64 (1.9.2-2ubuntu0.20.04.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libsystemd0_245.4-4ubuntu3.11_amd64.deb ...
Unpacking libsystemd0:amd64 (245.4-4ubuntu3.11) over (245.4-4ubuntu3.3) ...
Setting up libsystemd0:amd64 (245.4-4ubuntu3.11) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libudev1_245.4-4ubuntu3.11_amd64.deb ...
Unpacking libudev1:amd64 (245.4-4ubuntu3.11) over (245.4-4ubuntu3.3) ...
Setting up libudev1:amd64 (245.4-4ubuntu3.11) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libzstd1_1.4.4+dfsg-3ubuntu0.1_amd64.deb ...
Unpacking libzstd1:amd64 (1.4.4+dfsg-3ubuntu0.1) over (1.4.4+dfsg-3) ...
Setting up libzstd1:amd64 (1.4.4+dfsg-3ubuntu0.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libapt-pkg6.0_2.0.6_amd64.deb ...
Unpacking libapt-pkg6.0:amd64 (2.0.6) over (2.0.2ubuntu0.1) ...
Setting up libapt-pkg6.0:amd64 (2.0.6) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../gpgv_2.2.19-3ubuntu2.1_amd64.deb ...
Unpacking gpgv (2.2.19-3ubuntu2.1) over (2.2.19-3ubuntu2) ...
Setting up gpgv (2.2.19-3ubuntu2.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../ubuntu-keyring_2020.02.11.4_all.deb ...
Unpacking ubuntu-keyring (2020.02.11.4) over (2020.02.11.2) ...
Setting up ubuntu-keyring (2020.02.11.4) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libnettle7_3.5.1+really3.5.1-2ubuntu0.2_amd64.deb ...
Unpacking libnettle7:amd64 (3.5.1+really3.5.1-2ubuntu0.2) over (3.5.1+really3.5.1-2) ...
Setting up libnettle7:amd64 (3.5.1+really3.5.1-2ubuntu0.2) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libhogweed5_3.5.1+really3.5.1-2ubuntu0.2_amd64.deb ...
Unpacking libhogweed5:amd64 (3.5.1+really3.5.1-2ubuntu0.2) over (3.5.1+really3.5.1-2) ...
Setting up libhogweed5:amd64 (3.5.1+really3.5.1-2ubuntu0.2) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libp11-kit0_0.23.20-1ubuntu0.1_amd64.deb ...
Unpacking libp11-kit0:amd64 (0.23.20-1ubuntu0.1) over (0.23.20-1build1) ...
Setting up libp11-kit0:amd64 (0.23.20-1ubuntu0.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libgnutls30_3.6.13-2ubuntu1.6_amd64.deb ...
Unpacking libgnutls30:amd64 (3.6.13-2ubuntu1.6) over (3.6.13-2ubuntu1.3) ...
Setting up libgnutls30:amd64 (3.6.13-2ubuntu1.6) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libseccomp2_2.5.1-1ubuntu1~20.04.1_amd64.deb ...
Unpacking libseccomp2:amd64 (2.5.1-1ubuntu1~20.04.1) over (2.4.3-1ubuntu3.20.04.3) ...
Setting up libseccomp2:amd64 (2.5.1-1ubuntu1~20.04.1) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../archives/apt_2.0.6_amd64.deb ...
Unpacking apt (2.0.6) over (2.0.2ubuntu0.1) ...
Setting up apt (2.0.6) ...
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libpam0g_1.3.1-5ubuntu4.2_amd64.deb ...
Unpacking libpam0g:amd64 (1.3.1-5ubuntu4.2) over (1.3.1-5ubuntu4.1) ...
Setting up libpam0g:amd64 (1.3.1-5ubuntu4.2) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
(Reading database ... 4121 files and directories currently installed.)
Preparing to unpack .../libpam-modules-bin_1.3.1-5ubuntu4.2_amd64.deb ...
Unpacking libpam-modules-bin (1.3.1-5ubuntu4.2) over (1.3.1-5ubuntu4.1) ...
Setting up libpam-modules-bin (1.3.1-5ubuntu4.2) ...
(Reading database ... 4123 files and directories currently installed.)
Preparing to unpack .../libpam-modules_1.3.1-5ubuntu4.2_amd64.deb ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Unpacking libpam-modules:amd64 (1.3.1-5ubuntu4.2) over (1.3.1-5ubuntu4.1) ...
Setting up libpam-modules:amd64 (1.3.1-5ubuntu4.2) ...
(Reading database ... 4127 files and directories currently installed.)
Preparing to unpack .../libpam-runtime_1.3.1-5ubuntu4.2_all.deb ...
Unpacking libpam-runtime (1.3.1-5ubuntu4.2) over (1.3.1-5ubuntu4.1) ...
Setting up libpam-runtime (1.3.1-5ubuntu4.2) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.0 /usr/local/share/perl/5.30.0 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
(Reading database ... 4127 files and directories currently installed.)
Preparing to unpack .../passwd_1%3a4.8.1-1ubuntu5.20.04.1_amd64.deb ...
Unpacking passwd (1:4.8.1-1ubuntu5.20.04.1) over (1:4.8.1-1ubuntu5.20.04) ...
Setting up passwd (1:4.8.1-1ubuntu5.20.04.1) ...
(Reading database ... 4127 files and directories currently installed.)
Preparing to unpack .../libprocps8_2%3a3.3.16-1ubuntu2.2_amd64.deb ...
Unpacking libprocps8:amd64 (2:3.3.16-1ubuntu2.2) over (2:3.3.16-1ubuntu2) ...
Preparing to unpack .../procps_2%3a3.3.16-1ubuntu2.2_amd64.deb ...
Unpacking procps (2:3.3.16-1ubuntu2.2) over (2:3.3.16-1ubuntu2) ...
Setting up libprocps8:amd64 (2:3.3.16-1ubuntu2.2) ...
Setting up procps (2:3.3.16-1ubuntu2.2) ...
Processing triggers for libc-bin (2.31-0ubuntu9.2) ...
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  stress
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 18.4 kB of archives.
After this operation, 55.3 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu focal/universe amd64 stress amd64 1.0.4-6 [18.4 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 18.4 kB in 1s (13.6 kB/s)
Selecting previously unselected package stress.
(Reading database ... 4127 files and directories currently installed.)
Preparing to unpack .../stress_1.0.4-6_amd64.deb ...
Unpacking stress (1.0.4-6) ...
Setting up stress (1.0.4-6) ...
Removing intermediate container 746aa8bf0216
 ---> 24c27b440944
Successfully built 24c27b440944
Successfully tagged localhost:5000/stress-kata:latest
~/go/src/github.com/kata-containers/tests/integration/kubernetes
Device "cni0" does not exist.
Start crio service
[init.sh:99] INFO: Stop containerd service
Created symlink /etc/systemd/system/multi-user.target.wants/crio.service → /usr/local/lib/systemd/system/crio.service.
● crio.service - Container Runtime Interface for OCI (CRI-O)
     Loaded: loaded (/usr/local/lib/systemd/system/crio.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2021-08-24 10:10:14 UTC; 5s ago
       Docs: https://github.com/cri-o/cri-o
   Main PID: 68066 (crio)
      Tasks: 13
     Memory: 48.0M
     CGroup: /system.slice/crio.service
             └─68066 /usr/local/bin/crio

Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.337211739Z" level=debug msg="Usi…:1021"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.337339056Z" level=info msg="No s…go:56"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.337626999Z" level=info msg="AppA…go:33"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.338208031Z" level=info msg="Upda…o:371"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.338901152Z" level=debug msg="Rea…go:65"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.348614434Z" level=debug msg="Gol…o:367"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.348966932Z" level=debug msg="San…o:516"
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.349140937Z" level=debug msg="Reg…go:32"
Aug 24 10:10:14 ubuntu2004.localdomain systemd[1]: Started Container Runtime Interface for OCI (CRI-O).
Aug 24 10:10:14 ubuntu2004.localdomain crio[68066]: time="2021-08-24 10:10:14.374434033Z" level=debug msg="Mon…go:43"
Hint: Some lines were ellipsized, use -l to show in full.
Store custom stress image in registry
Unable to find image 'registry:2' locally
2: Pulling from library/registry
ddad3d7c1e96: Pull complete 
6eda6749503f: Pull complete 
363ab70c2143: Pull complete 
5b94580856e6: Pull complete 
12008541203a: Pull complete 
Digest: sha256:121baf25069a56749f249819e36b386d655ba67116d9c1c6c8594061852de4da
Status: Downloaded newer image for registry:2
aeb36b32a0aed52eb94b7dd990de1b07d4f5ec3ad114e21525d6ec7a5d8ec4e2
The push refers to repository [localhost:5000/stress-kata]
88c30ed9cfe5: Pushed 
f6253634dc78: Pushed 
9069f84dbbe9: Pushed 
bacd3af13903: Pushed 
latest: digest: sha256:6b3b6e7263c024b0756b886397a475531d96e5bdfaca12f981fbd38b8d59fa83 size: 1155
Init cluster using /var/run/crio/crio.sock
[init] Using Kubernetes version: v1.21.1
[preflight] Running pre-flight checks
[preflight] Pulling images required for setting up a Kubernetes cluster
[preflight] This might take a minute or two, depending on the speed of your internet connection
[preflight] You can also perform this action in beforehand using 'kubeadm config images pull'
[certs] Using certificateDir folder "/etc/kubernetes/pki"
[certs] Generating "ca" certificate and key
[certs] Generating "apiserver" certificate and key
[certs] apiserver serving cert is signed for DNS names [kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local ubuntu2004.localdomain] and IPs [10.96.0.1 192.168.121.114]
[certs] Generating "apiserver-kubelet-client" certificate and key
[certs] Generating "front-proxy-ca" certificate and key
[certs] Generating "front-proxy-client" certificate and key
[certs] Generating "etcd/ca" certificate and key
[certs] Generating "etcd/server" certificate and key
[certs] etcd/server serving cert is signed for DNS names [localhost ubuntu2004.localdomain] and IPs [192.168.121.114 127.0.0.1 ::1]
[certs] Generating "etcd/peer" certificate and key
[certs] etcd/peer serving cert is signed for DNS names [localhost ubuntu2004.localdomain] and IPs [192.168.121.114 127.0.0.1 ::1]
[certs] Generating "etcd/healthcheck-client" certificate and key
[certs] Generating "apiserver-etcd-client" certificate and key
[certs] Generating "sa" key and public key
[kubeconfig] Using kubeconfig folder "/etc/kubernetes"
[kubeconfig] Writing "admin.conf" kubeconfig file
[kubeconfig] Writing "kubelet.conf" kubeconfig file
[kubeconfig] Writing "controller-manager.conf" kubeconfig file
[kubeconfig] Writing "scheduler.conf" kubeconfig file
[kubelet-start] Writing kubelet environment file with flags to file "/var/lib/kubelet/kubeadm-flags.env"
[kubelet-start] Writing kubelet configuration to file "/var/lib/kubelet/config.yaml"
[kubelet-start] Starting the kubelet
[control-plane] Using manifest folder "/etc/kubernetes/manifests"
[control-plane] Creating static Pod manifest for "kube-apiserver"
[control-plane] Creating static Pod manifest for "kube-controller-manager"
[control-plane] Creating static Pod manifest for "kube-scheduler"
[etcd] Creating static Pod manifest for local etcd in "/etc/kubernetes/manifests"
[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory "/etc/kubernetes/manifests". This can take up to 4m0s
[apiclient] All control plane components are healthy after 17.025995 seconds
[upload-config] Storing the configuration used in ConfigMap "kubeadm-config" in the "kube-system" Namespace
[kubelet] Creating a ConfigMap "kubelet-config-1.21" in namespace kube-system with the configuration for the kubelets in the cluster
[upload-certs] Skipping phase. Please see --upload-certs
[mark-control-plane] Marking the node ubuntu2004.localdomain as control-plane by adding the labels: [node-role.kubernetes.io/master(deprecated) node-role.kubernetes.io/control-plane node.kubernetes.io/exclude-from-external-load-balancers]
[mark-control-plane] Marking the node ubuntu2004.localdomain as control-plane by adding the taints [node-role.kubernetes.io/master:NoSchedule]
[bootstrap-token] Using token: iiydnw.uiwxms8dnrqgadlo
[bootstrap-token] Configuring bootstrap tokens, cluster-info ConfigMap, RBAC Roles
[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to get nodes
[bootstrap-token] configured RBAC rules to allow Node Bootstrap tokens to post CSRs in order for nodes to get long term certificate credentials
[bootstrap-token] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap Token
[bootstrap-token] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
[bootstrap-token] Creating the "cluster-info" ConfigMap in the "kube-public" namespace
[kubelet-finalize] Updating "/etc/kubernetes/kubelet.conf" to point to a rotatable kubelet client certificate and key
[addons] Applied essential addon: CoreDNS
[addons] Applied essential addon: kube-proxy

Your Kubernetes control-plane has initialized successfully!

To start using your cluster, you need to run the following as a regular user:

  mkdir -p $HOME/.kube
  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
  sudo chown $(id -u):$(id -g) $HOME/.kube/config

Alternatively, if you are the root user, you can run:

  export KUBECONFIG=/etc/kubernetes/admin.conf

You should now deploy a pod network to the cluster.
Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
  https://kubernetes.io/docs/concepts/cluster-administration/addons/

Then you can join any number of worker nodes by running the following on each as root:

kubeadm join 192.168.121.114:6443 --token iiydnw.uiwxms8dnrqgadlo \
	--discovery-token-ca-cert-hash sha256:d3fd825912be219ec48e6b173b71159aec2a641ee204f03c5c196fc931252a25 
Kubelet options:
KUBELET_KUBEADM_ARGS="--allowed-unsafe-sysctls=kernel.msg*,kernel.shm.*,net.* --container-runtime=remote --container-runtime-endpoint=/var/run/crio/crio.sock --pod-infra-container-image=k8s.gcr.io/pause:3.4.1 --v=4"
NAME                     STATUS     ROLES                  AGE   VERSION
ubuntu2004.localdomain   NotReady   control-plane,master   11s   v1.21.1
No resources found in default namespace.
Warning: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
podsecuritypolicy.policy/psp.flannel.unprivileged created
clusterrole.rbac.authorization.k8s.io/flannel created
clusterrolebinding.rbac.authorization.k8s.io/flannel created
serviceaccount/flannel created
configmap/kube-flannel-cfg created
daemonset.apps/kube-flannel-ds created
No resources found
kube-system   kube-apiserver-ubuntu2004.localdomain            1/1     Running    0          5s
kube-system   kube-controller-manager-ubuntu2004.localdomain   1/1     Running    0          11s
kube-system   etcd-ubuntu2004.localdomain                      1/1     Running    0          11s
kube-system   kube-scheduler-ubuntu2004.localdomain            1/1     Running    0          11s
kube-system   coredns-558bd4d5db-8l8w5                         1/1     Running   0          40s
kube-system   coredns-558bd4d5db-m7k75                         1/1     Running   0          40s
Create kata RuntimeClass resource
runtimeclass.node.k8s.io/kata created
node/ubuntu2004.localdomain untainted
1..1
ok 1 Running with postStart and preStop handlers
1..1
not ok 1 Block Storage Support
# (from function `setup' in test file k8s-block-volume.bats, line 23)
#   `sudo truncate "$tmp_disk_image" --size "$vol_capacity"' failed
# [bats-exec-test:23] INFO: k8s configured to use runtimeclass
# truncate: cannot open '/tmp/disk.pUYqmh.img' for writing: Permission denied
# Error from server (NotFound): pods "pod-block-pv" not found
# Error from server (NotFound): pods "pod-block-pv" not found
# Error from server (NotFound): persistentvolumeclaims "block-loop-pvc" not found
# Error from server (NotFound): persistentvolumes "block-loop-pv" not found
# Error from server (NotFound): storageclasses.storage.k8s.io "local-storage" not found
# losetup: : failed to use device: No such device
[run_kubernetes_tests.sh:65] ERROR: bats k8s-block-volume.bats
[reset] Reading configuration from the cluster...
[reset] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -o yaml'
[preflight] Running pre-flight checks
[reset] Removing info for node "ubuntu2004.localdomain" from the ConfigMap "kubeadm-config" in the "kube-system" Namespace
[reset] Stopping the kubelet service
[reset] Unmounting mounted directories in "/var/lib/kubelet"
[reset] Deleting contents of config directories: [/etc/kubernetes/manifests /etc/kubernetes/pki]
[reset] Deleting files: [/etc/kubernetes/admin.conf /etc/kubernetes/kubelet.conf /etc/kubernetes/bootstrap-kubelet.conf /etc/kubernetes/controller-manager.conf /etc/kubernetes/scheduler.conf]
[reset] Deleting contents of stateful directories: [/var/lib/etcd /var/lib/kubelet /var/lib/dockershim /var/run/kubernetes /var/lib/cni]

The reset process does not clean CNI configuration. To do so, you must remove /etc/cni/net.d

The reset process does not reset or clean up iptables rules or IPVS tables.
If you wish to reset iptables, you must do so manually by using the "iptables" command.

If your cluster was setup to utilize IPVS, run ipvsadm --clear (or similar)
to reset your system's IPVS tables.

The reset process does not clean your kubeconfig files and you must remove them manually.
Please, check the contents of the $HOME/.kube/config file.
kata-registry
Not /run/vc/sbs directory found
make: *** [Makefile:117: kubernetes] Error 1
```

--- a/integration/kubernetes/k8s-block-volume.bats
+++ b/integration/kubernetes/k8s-block-volume.bats
@@ -20,7 +20,7 @@ setup() {
 
 	# Create Loop Device
 	tmp_disk_image=$(mktemp --tmpdir disk.XXXXXX.img)
-	sudo truncate "$tmp_disk_image" --size "$vol_capacity"
+	truncate "$tmp_disk_image" --size "$vol_capacity"
 	loop_dev=$(sudo losetup -f)
 	sudo losetup "$loop_dev" "$tmp_disk_image"
 }

